src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -235,6 +235,24 @@ public enum LoadingStatusType
             Failed
         }
 
+        /// <summary>
+        /// Enumeration for MaskingMode of image.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum MaskingModeType
+        {
+            /// <summary>
+            /// Loading preparing status.
```

Comment:
I think this comment is copy-paste. We need to write something more good sentence.

Reply:
Thank you I fixned